defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 es_version: "7.5.1"
 es_use_snapshot_release: false
-es_enable_xpack: true
+oss_version: false
```

Comment:
Could/should we add some kind of backwards compatibility so that if `es_enable_xpack` is set it sets `oss_version` automatically and logs a deprecation message? Perhaps also a deprecation message (or straight up failure) if  `es_xpack_features` is set with instructions on how to achieve this instead (which I think would be settings in `es_config`)?